spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -10,35 +10,41 @@
       )
 
       job.perform(Time.zone.now)
-    end
 
-    response = JSON.parse(response_json)
-    expect(job_analytics).to have_logged_event(
-      'GetUspsProofingResultsJob: Enrollment status updated',
-      assurance_level: response['assuranceLevel'],
-      enrollment_code: pending_enrollment.enrollment_code,
-      enrollment_id: pending_enrollment.id,
-      failure_reason: response['failureReason'],
-      fraud_suspected: response['fraudSuspected'],
-      issuer: pending_enrollment.issuer,
-      minutes_since_last_status_check: 15.0,
-      minutes_since_last_status_update: 2.days.in_minutes,
-      minutes_to_completion: 3.days.in_minutes,
-      minutes_since_established: 3.days.in_minutes,
-      passed: passed,
-      primary_id_type: response['primaryIdType'],
-      proofing_city: response['proofingCity'],
-      proofing_post_office: response['proofingPostOffice'],
-      proofing_state: response['proofingState'],
-      reason: anything,
-      response_message: response['responseMessage'],
-      response_present: true,
-      scan_count: response['scanCount'],
-      secondary_id_type: response['secondaryIdType'],
-      status: response['status'],
-      transaction_end_date_time: anything,
-      transaction_start_date_time: anything,
-    )
+      response = JSON.parse(response_json)
+      expect(job_analytics).to have_logged_event(
+        'GetUspsProofingResultsJob: Enrollment status updated',
+        assurance_level: response['assuranceLevel'],
+        enrollment_code: pending_enrollment.enrollment_code,
+        enrollment_id: pending_enrollment.id,
+        failure_reason: response['failureReason'],
+        fraud_suspected: response['fraudSuspected'],
+        issuer: pending_enrollment.issuer,
+        minutes_since_last_status_check: 15.0,
+        minutes_since_last_status_update: 2.days.in_minutes,
+        minutes_to_completion: range_approximating(
+          3.days.in_minutes, vary_left: -60,
+                             vary_right: 5
+        ),
+        minutes_since_established: range_approximating(
+          3.days.in_minutes, vary_left: -60,
+                             vary_right: 5
+        ),
```

Comment:
```suggestion
        minutes_to_completion: 3.days.in_minutes,
        minutes_since_established: 3.days.in_minutes,
```
since time is frozen, you should be able to do this. I tried it locally and ran the test 50 times in a row with no failures.

Reply:
Yeah just passed 200 times without issue. removing.